indy-wrapper/src/main/java/nl/quintor/studybits/indy/wrapper/dto/CredentialInfo.java

```diff
@@ -7,16 +7,24 @@
 
 import java.util.Map;
 
-@AllArgsConstructor
 @Data
+@AllArgsConstructor
 @NoArgsConstructor
-public class ClaimReferent implements Serializable {
+public class CredentialInfo {
+    @JsonProperty( "schema_id" )
+    private String schemaId;
+
+    @JsonProperty( "cred_def_id" )
+    private String credDefId;
+
     private String referent;
+
+    @JsonProperty( "attrs" )
     private Map<String, String> attrs;
-    @JsonProperty( "schema_key" )
-    private SchemaKey schemaKey;
-    @JsonProperty( "issuer_did" )
-    private String issuerDid;
-    @JsonProperty( "revoc_reg_seq_no" )
-    private Integer revocRegSeqNo;
-}
+
+    @JsonProperty( "rev_reg_id" )
+    private Integer revRegId;
```

Comment:
What is `revRegId`? I assume `credRevId` is `Credential Reference Id`? These variable names are not too informative. Consider renaming.